shared/src/commonMain/kotlin/org/hyperskill/app/study_plan/widget/view/model/StudyPlanWidgetViewState.kt

```diff
@@ -47,13 +47,12 @@ sealed interface StudyPlanWidgetViewState {
         val hypercoinsAward: Int?
     ) {
         val isClickable: Boolean
-            get() = state == SectionItemState.NEXT
+            get() = state == SectionItemState.NEXT || state == SectionItemState.IDLE
```

Comment:
The user should be able to click on any learning activity (like on the web), so let's delete `isClickable` property and all code that disables clicks on the platforms